### ISSUE TITLE
Augmented Sextant crashes on use

### ISSUE THREAD

**Issue by RandomSomething98** [HellFirePvP/AstralSorcery#1062]
With AS version 1.10.11, on a server running forge-1.12.2-14.23.5.2814, when I open up an augmented sextant (where I look doesn't matter) the server crashes seconds later without a crash log. I haven't tested this on single player yet.

**Comment by Doomgull** [comment]
Going to need a log of some sort to do anything about this.

I'd also recommend updating to 1.10.13.

**Comment by HellFirePvP** [comment]
Did adjust some things serversided regarding this.
Update to 1.10.14 and test this again.

**Comment by Doomgull** [comment]
@RandomSomething98 any update?

**Comment by RandomSomething98** [comment]
The latest Astral Sorcery update fixed this, sorry for the belated response!

**Comment by RandomSomething98** [comment]
So I've been able to utilize the sextant a little bit on the latest update, but it still mysteriously crashes. One thing I have noticed is that the times it has crashed I pressed shift quickly to get to the selections on the right, but the crash would interrupt that process. I looked into my server logs and my start.sh file literally just restarts, theres no error or indication of what caused it. Could this be caused by what's on the JVM arguments in my start.sh conflicting with Astral Sorcery? I'll post the start.sh file here: https://pastebin.com/Pf7rrigC

**Comment by Doomgull** [comment]
Anything in the os event logs?

**Comment by RandomSomething98** [comment]
Sorry, the server is on a VPS running Debian, and I can't find the OS event logs.

**Comment by Doomgull** [comment]
I meant from your workstation, the client

**Comment by Doomgull** [comment]
@RandomSomething98 can you try the latest AS release (1.17) and let us know if still happening? Client log when it happens would be good.

**Comment by RandomSomething98** [comment]
Sorry for the wait.
I've been able to replicate this error by spawning in the world then using the sextant within the first 10 seconds or so I'm in. I've looked into it client side, and found this comment by the mod foamfix.

[14:11:00] [Client thread/INFO] [foamfix]: Unloaded world <net.minecraft.client.multiplayer.WorldClient@4a4ddb12 0 MultiplayerChunkCache: 441, 441> is still hanging around after 60 seconds.
[14:11:00] [Client thread/INFO] [foamfix]: This may be due to a world leak, or there is little memory pressure.
[14:11:00] [Client thread/INFO] [foamfix]: Use the relevant FoamFix options to adjust the wait time.
[14:11:44] [Client thread/INFO] [foamfix]: Unloaded world 

So I think this may just be a foamfix issue. Googling this tells me that perhaps my server isn't beefy enough (I'm running a custom modpack with 53 mods on 2 gigs).
In trying to replicate this, I came across a new crash. server side when using the sextant like before, which I have linked here: https://pastebin.com/MbF9sAZd
